tests/acceptance/test_emails.py

```diff
@@ -20,177 +42,11 @@ def build_url(self, path, format='html'):
             }),
         )
 
-    def test_assigned_html(self):
```

Comment:
noticed the other day how slow these tests were (thanks Zeus!) and combining them will drastically increase their speed